quantecon/game_theory/mclennan_tourky.py

```diff
@@ -6,7 +6,7 @@
 """
 import numbers
 import numpy as np
-from ..compute_fp import _compute_fixed_point_ig
+from quantecon._compute_fp import _compute_fixed_point_ig
```

Comment:
I think this should be `from .._compute_fp`?

```suggestion
from .._compute_fp import _compute_fixed_point_ig
```

(Several other places.)

Reply:
Relative imports are often confusing so IMHO it's better to use absolute imports.

Reply:
Is it just a matter of exposition? I thought it made some real difference in some occasions (but I don't know what it would be).

But one general idea is that one should avoid hard coding: If we wanted to change the library name (although it is very unlikely to happen), the part would have to be modified.

Reply:
@mmcky You made some argument about this in https://github.com/QuantEcon/QuantEcon.py/issues/137#issuecomment-97999281 (long time ago..). Would that apply here?